pkg/apiserver/apiserver.go

```diff
@@ -212,6 +213,7 @@ func newAPIHandlerEngine() (apiHandlerEngine *gin.Engine, endpoint *gin.RouterGr
 	apiHandlerEngine.Use(apiutils.MWHandleErrors())
 
 	endpoint = apiHandlerEngine.Group("/dashboard/api")
+	endpoint.StaticFS("/speedscope", speedscopeFiles.Assets())
```

Comment:
I think we should not expose speedscope static files in the `/api` category. How about letting it to be `/dashboard/speedscope`?